project/MimaExcludes.scala

```diff
@@ -60,6 +60,10 @@ object MimaExcludes {
               "org.apache.spark.ml.regression.LeastSquaresCostFun.this"),
             ProblemFilters.exclude[MissingMethodProblem](
               "org.apache.spark.ml.classification.LogisticCostFun.this"),
+            ProblemFilters.exclude[MissingMethodProblem](
```

Comment:
I am also learning so this is only my two cents: the migration manager (MiMa) will check for binary incompatibility from previous versions. In this PR, the `add` and `merge` method of `LogisticAggregator` is revised to take different argument type than before. A library written for previous versions of `LogisticAggregator` will not be able to link to the new version. MiMa will detect this binary incompatibility. 

We stop MiMa from complaining by excluding these methods from the compatibility checking.